access/src/main/java/edu/unc/lib/dl/ui/controller/SearchActionController.java

```diff
@@ -101,10 +101,10 @@ public String list(Model model, HttpServletRequest request) {
 	public String listContents(@PathVariable("pid") String pid, Model model, HttpServletRequest request) {
 		SearchRequest searchRequest = generateSearchRequest(request);
 		searchRequest.getSearchState().setResourceTypes(
-				Arrays.asList(ResourceType.Aggregate.name(), ResourceType.File.name()));
+				Arrays.asList(ResourceType.Aggregate.name()));
```

Comment:
Sorry, thought I'd already left a comment on this, but probably losing track of things at the conference.
We still want Files to show up when they aren't in works in fcrepo3 land.  What we want instead is to suppress children of works.  You can do this by calling `searchState.setIncludeParts(false)`.  Having rollup turned off is also good.